sdk/storage/azure-storage-blob-cryptography/src/main/java/com/azure/storage/blob/specialized/cryptography/EncryptedBlobClientBuilder.java

```diff
@@ -73,6 +73,7 @@
 import java.util.regex.Pattern;
 
 import static com.azure.storage.blob.implementation.util.BuilderHelper.createTracer;
+import static com.azure.storage.blob.specialized.cryptography.CryptographyConstants.GCM_ENCRYPTION_REGION_LENGTH;
```

Comment:
We should be able to remove this import now that the default is in the options class.